README.md

```diff
@@ -0,0 +1,39 @@
+# Manjaro Arkdep variants
+
+This repository contains Manjaro Immutable variant configuration for arkdep-build.
+
+Variants are recipes used by arkdep-build to create installable images in Arkdep format.
+
+## System requirements
+
+- Have arkdep-build and dependencies installed.
+
+```bash
+sudo pacman -Sy arkdep
+```
+
+## Building a variant
+
+First clone the repo and navigate to its root;
```

Comment:
. or :

Reply:
I see you later used semicolon again. Not an accident?

Reply:
That is my personal writing style. Means "Look at thing below". I can change that.